examples/sinatra/server.rb

```diff
@@ -23,8 +23,10 @@ def db_store_identity(user_id, identity); EXAMPLE_DB[user_id][:identity] = ident
     def db_load_identity(user_id); EXAMPLE_DB[user_id]&.fetch(:identity, nil); end
   end
 
-  # TODO: ensure config is stored in a secure place
   configure do
+    enable :logging
```

Comment:
Hm, is this required by the Sinatra upgrade?

(It's fine, but just nitpicking: in case it's not really required by the bump, it's nice to have potential behavior changes in other commits!)

Reply:
You're right, I've extracted this in another commit.